pwa/src/components/DiagnosisForm.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement } from 'react'
+import { List, Checkbox, InputItem, WhiteSpace } from 'antd-mobile'
+import '../scss/DiagnosisForm.scss'
+import * as data from './DiagnosisFields.json'
+
+const CheckboxItem = Checkbox.CheckboxItem
+
+/*
+
+Diagnosis Fields are stored in DiagnosisFields.json in the format:
+
+{
+    "diagnoses" : [
+        {
+            "name": "Name",
+            "id": "id",
+            "text-input-title": "Title",
+            "text-input-type": "type" (i.e. number)
+            "group-title": "Group Title",
+            "group": [
+                { "name": "Name1", "id": "id1" },
+                { "name": "Name2", "id": "id2" }
+            ]
+        },
+        {...}
+    ]
+}
+*/
```

Comment:
Is this comment valuable to have in this file and DIagnosisForm.test? If not, it can be removed.

Reply:
I vote to keep it 👍 I'll add 'max-length' to it since I added that to the JSON?

Again lmk if there are some concerns on having max-length in the JSON body, not trying to make an executive decision on that, just would be a very easy rollback if we decide otherwise.